src/github.com/t3chguy/riot-static/utils.go

```diff
@@ -13,3 +13,16 @@
 // limitations under the License.
 
 package main
+
+import "strconv"
+
+func calcPaginationPage(pageString string, size int) (page int, skip int, end int) {
```

Comment:
What is this supposed to do? Some docs would be nice. Is `size` the number of elements per page? The number of pages? What's `skip` and `end`?